Jenkinsfile

```diff
@@ -69,6 +69,7 @@ for(j = 0; j < jdks.size(); j++) {
                                              allowEmptyArchive: true, // in case we forgot to reincrementalify
                                              fingerprint: true
                         }
+                        publishHTML([allowMissing: true, alwaysLinkToLastBuild: false, includes: 'japicmp.html', keepAll: false, reportDir: 'core/target/japicmp', reportFiles: 'japicmp.html', reportName: 'API compatibility', reportTitles: 'japicmp report'])
```

Comment:
Do we want to merge this in?

Reply:
⇒ https://github.com/jenkinsci/jenkins/pull/4848#discussion_r518862224